packages/places/__tests__/getPlaces.test.js

```diff
@@ -0,0 +1,10 @@
+const { getPlaces } = require('../src/index');
+
+describe('given a client that tries to get the places', () => {
+    describe('when the client does the GET request', () => {
+        test('then the db returns an array with the places', async () => {
+            const data = await getPlaces();
+            expect(typeof data).toBe('object');
+        });
+    });
+});
```

Comment:
@Alejandrocuartas are these tests using the real connection? or what the tests are doing? based on the code what I'm seeing is that the `getPlaces` receives a model and it returns a function that is the responsible to execute the `findAll`, so this call should be tested either as the possible error cases

Reply:
Yes, im using a real connection to a local database.

Reply:
@Alejandrocuartas unit tests should not connect agains the DB, instead, what we need to do is to mock the connection.